app/view/showCompany/showCompany.js

```diff
@@ -30,4 +31,59 @@ angular.module('csrLookupApp.showCompany', ['ngRoute'])
   }, function errorCallback(response) {
     $scope.evidence_records = null
   });
+  
+   function fillWikipediaBox(companyWikipediaName)
+   {
+       var wikiData;
+        $.ajax({
```

Comment:
I think mostly we use 2 spaces for indentation. Could you update this to match that convention?